showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITPagination.java

```diff
@@ -0,0 +1,167 @@
+package com.google.showcase.v1beta1.it;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.api.gax.rpc.ServerStream;
+import com.google.showcase.v1beta1.EchoClient;
+import com.google.showcase.v1beta1.EchoResponse;
+import com.google.showcase.v1beta1.ExpandRequest;
+import com.google.showcase.v1beta1.PagedExpandRequest;
+import com.google.showcase.v1beta1.it.util.TestClientInitializer;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ITPagination {
+
+  private static EchoClient grpcClient;
+  private static EchoClient httpjsonClient;
+
+  @BeforeClass
+  public static void createClients() throws Exception {
+    // Create gRPC Echo Client
+    grpcClient = TestClientInitializer.createGrpcEchoClient();
+    // Create Http JSON Echo Client
+    httpjsonClient = TestClientInitializer.createHttpJsonEchoClient();
+  }
+
+  @AfterClass
+  public static void destroyClients() throws InterruptedException {
+    grpcClient.close();
+    grpcClient.awaitTermination(TestClientInitializer.AWAIT_TERMINATION_SECONDS, TimeUnit.SECONDS);
+    httpjsonClient.close();
+    httpjsonClient.awaitTermination(
+        TestClientInitializer.AWAIT_TERMINATION_SECONDS, TimeUnit.SECONDS);
+  }
+
+  @Test
+  public void testExpandGrpc() {
+    String content = "Testing the entire response is the same";
+    int expected = content.split(" ").length;
+    ServerStream<EchoResponse> echoResponses =
+        grpcClient.expandCallable().call(ExpandRequest.newBuilder().setContent(content).build());
+    List<String> values = new ArrayList<>();
+    for (EchoResponse echoResponse : echoResponses) {
+      values.add(echoResponse.getContent());
+    }
+
+    String response = String.join(" ", values);
+    assertThat(values.size()).isEqualTo(expected);
+    assertThat(response).isEqualTo(content);
+  }
+
+  @Test
+  public void testExpandHttpJson() {
+    String content = "Testing the entire response is the same";
+    int expected = content.split(" ").length;
+    ServerStream<EchoResponse> echoResponses =
+        httpjsonClient
+            .expandCallable()
+            .call(ExpandRequest.newBuilder().setContent(content).build());
+    List<String> values = new ArrayList<>();
+    for (EchoResponse echoResponse : echoResponses) {
+      values.add(echoResponse.getContent());
+    }
+
+    String response = String.join(" ", values);
+    assertThat(values.size()).isEqualTo(expected);
+    assertThat(response).isEqualTo(content);
```

Comment:
To prevent the assert statements from containing too much logic, I wonder if we could simplify them by hard-coding the expected values? In other words:

```
 assertThat(values.size()).isEqualTo(7);
  assertThat(values).containsExactly("Testing", "the", ...)).inOrder();
```
This also makes the expected values more visible.

Reply:
Updating to use `assertThat(values).containsExactly("Testing", "the", ...)).inOrder();`. I think this should have less logic inside the test.